src/resources/postcss/base/forms/_tabs.pcss

```diff
@@ -68,18 +69,36 @@
 			&:first-child label {
 				@mixin form-control-select-list-item-first;
 
+				body#top & { /* Enfold compatibility */
+					@mixin form-control-select-list-item-first;
+				}
+
 				@media (--viewport-medium) {
 					margin-top: -6px;
 					padding-top: 17px;
+
+					body#top & { /* Enfold compatibility */
```

Comment:
Let's keep media query overrides within the same selector, so for example:

```css
body#top & {
    @mixin form-control-select-list-item-first;

    @media (--viewport-medium) {
        margin-top: -6px;
        padding-top: 17px;
    }
}
```

This keeps all styles packaged together rather than spread apart in media queries.